### DISCUSSION TITLE
请问侧栏链接从新窗口打开是需要修改哪个文件啊？

### DISCUSSION THREAD

**Discussion by dianso** [gaowei-space/markdown-blog#62] [General]
请问侧栏链接从新窗口打开是需要修改哪个文件啊？

**Comment by gaowei-space** [comment]
试试把 `layout.html` 文件中 `<a href="{{.Link}}">` 改为 `<a href="{{.Link}}" target="_blank">` 看看

**Reply by dianso** [reply]
感谢，这样可以，请问需要修改吗？需要了我提交个pr

侧边栏新窗口打开我之前都是用js来完成

// 获取所有具有类名为"chapter"的<li>元素
var chapters = document.querySelectorAll('li.chapter');

// 遍历每个<li>元素
chapters.forEach(function(chapter) {
  // 获取当前<li>元素下的链接元素
  var link = chapter.querySelector('a');

  // 修改链接的目标属性为 "_blank"
  link.target = '_blank';
});

**Reply by gaowei-space** [reply]
当然可以，如果是个配置项就更好了
